app/styles/ui/dialogs/_open-pull-request.scss

```diff
@@ -1,4 +1,7 @@
 .open-pull-request {
+  width: 850px;
```

Comment:
Fits in minimum window constraints -> may change this when solving for a responsive height/width design.